diff --git a/dist/rehype-pretty-code.js b/dist/rehype-pretty-code.js
index 2fe5768c15a210c17eaad74d0e6b6bd4f92a5de2..6e6689cf5498d208a58c0b5214886512a82bd596 100644
--- a/dist/rehype-pretty-code.js
+++ b/dist/rehype-pretty-code.js
@@ -11522,7 +11522,12 @@ function rehypePrettyCode(options = {}) {
         if (!isText(textElement)) {
           return;
         }
-        const strippedValue = textElement.value.replace(/\n$/, '');
+        const strippedValue = textElement.value.replace(/\n$/, '')
+            .replaceAll(/&lt;/g, '<')
+            .replaceAll(/&gt;/g, '>')
+            .replaceAll(/&#123;/g, '{')
+            .replaceAll(/&#125;/g, '}')
+            .replaceAll(/&amp;/g, '&');
         const trees = {};
         for (const [mode, highlighter] of highlighters.entries()) {
           try {
