diff --git a/dist/index.esm.js b/dist/index.esm.js
index d3715da1421eb47b00020cc190e6c500f1d8e247..1d4b4f34dbf6508c74a77e246aaf4d655d845553 100644
--- a/dist/index.esm.js
+++ b/dist/index.esm.js
@@ -2619,13 +2619,15 @@ function renderToHtml(lines, options = {}) {
 }
 const htmlEscapes = {
   "&": "&amp;",
-  "<": "&lt;",
-  ">": "&gt;",
+  "<": "{ '<' }",
+  ">": "{ '>' }",
+  "{": "{ '{' }",
+  "}": "{ '}' }",
   '"': "&quot;",
   "'": "&#39;"
 };
 function escapeHtml(html) {
-  return html.replace(/[&<>"']/g, (chr) => htmlEscapes[chr]);
+  return html.replace(/[&<>{}"']/g, (chr) => htmlEscapes[chr]);
 }
 function getLineClasses(lineOptions) {
   const lineClasses = /* @__PURE__ */ new Set(["line"]);
diff --git a/dist/index.js b/dist/index.js
index 55e732b398923b80f06644dee73d07153f0d3174..d6873ed67d39a751500ad1ec2250346a8e86308f 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2621,13 +2621,15 @@ function renderToHtml(lines, options = {}) {
 }
 const htmlEscapes = {
   "&": "&amp;",
-  "<": "&lt;",
-  ">": "&gt;",
+  "<": "{ '<' }",
+  ">": "{ '>' }",
+  "{": "{ '{' }",
+  "}": "{ '}' }",
   '"': "&quot;",
   "'": "&#39;"
 };
 function escapeHtml(html) {
-  return html.replace(/[&<>"']/g, (chr) => htmlEscapes[chr]);
+  return html.replace(/[&<>{}"']/g, (chr) => htmlEscapes[chr]);
 }
 function getLineClasses(lineOptions) {
   const lineClasses = /* @__PURE__ */ new Set(["line"]);